### ISSUE TITLE
Linux CCompilerInvoker not working if locale is not english

### ISSUE THREAD

**Issue by XakepSDK** [oracle/graal#2416]
**Describe the issue**
A clear and concise description of the issue. We recommend using the [latest snapshot builds](https://github.com/graalvm/graalvm-ce-dev-builds/releases) to replicate the issue as  bugs are constantly being fixed in the master branch

**Steps to reproduce the issue**
Please include both build steps as well as run steps
1. Set locale to not English or C, German for example
2. Try to run native-image

**Describe GraalVM and your environment:**
 - GraalVM version or commit id if built from source:
```
openjdk version "11.0.5" 2019-10-15
OpenJDK Runtime Environment (build 11.0.5+10-jvmci-19.3-b05-LTS)
OpenJDK 64-Bit Server VM GraalVM 20.1.0-dev (build 11.0.5+10-jvmci-19.3-b05-LTS, mixed mode, sharing)
```
https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1.0-ea+26.zip
 - JDK major version: 11
 - OS: Arch Linux
 - Architecture: AMD64

**More details**
Native image fails to find C compiler because `gcc -v` returns localized output.
Fix: add env. variable `LC_ALL=C` before executing command
https://github.com/oracle/graal/blob/master/substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/codegen/CCompilerInvoker.java#L320

**Comment by olpaw** [comment]
> Fix: add env. variable LC_ALL=C before executing command

@XakepSDK thanks for the hint!

**Comment by olpaw** [comment]
The fix is on master and will also be added to 20.1.
